server/data/data-api/src/main/java/org/apache/james/dlp/api/DLPConfigurationLoader.java

```diff
@@ -19,10 +19,12 @@
 
 package org.apache.james.dlp.api;
 
+import java.util.Optional;
 import java.util.stream.Stream;
 
 import org.apache.james.core.Domain;
 
 public interface DLPConfigurationLoader {
     Stream<DLPConfigurationItem> list(Domain domain);
+    Optional<DLPConfigurationItem> fetch(Domain domain, DLPConfigurationItem.Id ruleId);
```

Comment:
should be part of DLPConfigurationStore, Loader is, well, to load the configuration.